src/components/InputField/component.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getIn } from 'formik';
+import { Form, Input, Icon } from 'antd';
+
+const InputField = ({
+  placeholder,
+  autoComplete,
+  type,
+  icon,
+  field,
+  form: { touched, errors },
+  ...props
+}) => {
+  const fieldTouched = getIn(touched, field.name);
+  const fieldErrors = getIn(errors, field.name);
+
+  return (
+    <Form.Item
+      validateStatus={fieldErrors && fieldTouched && 'error'}
+      help={fieldErrors && fieldTouched && fieldErrors}
+    >
+      <Input
+        type={type}
+        autoComplete={autoComplete}
+        placeholder={placeholder}
+        prefix={(
+          icon && (
+            <Icon
+              type={icon}
+              style={{ color: 'rgba(0,0,0,.25)' }}
+            />
+          )
+        )}
+        {...{ ...field, ...props }}
```

Comment:
Just use construction without wrapping it with additional object, like
```
{...field}
{...props}
```

This is more readable

Reply:
Done.